benchmark/delete-reservation.sql

```diff
@@ -0,0 +1,100 @@
+DELETE FROM reservations WHERE id = 9103635;
```

Comment:
If you have to do this again, I can show you how to do a sql script to automate it.

Reply:
Thank you, @burkegg!